src/main/java/seedu/moneygowhere/ui/MainWindow.java

```diff
@@ -132,12 +132,16 @@ void fillInnerParts() {
      * Sets the default size based on {@code guiSettings}.
      */
     private void setWindowDefaultSize(GuiSettings guiSettings) {
+        /*
         primaryStage.setHeight(guiSettings.getWindowHeight());
         primaryStage.setWidth(guiSettings.getWindowWidth());
         if (guiSettings.getWindowCoordinates() != null) {
             primaryStage.setX(guiSettings.getWindowCoordinates().getX());
             primaryStage.setY(guiSettings.getWindowCoordinates().getY());
         }
+         */
```

Comment:
i think just delete this

Reply:
I am keeping the code here in case we decide to have a settings page, which utilises the preferences.json file